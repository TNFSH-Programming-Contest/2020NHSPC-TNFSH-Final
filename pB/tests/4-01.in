85
690.193 -479.895
678.135 -561.288
566.597 -500.997
580.162 -579.375
501.785 -573.346
513.843 -491.954
394.769 -345.749
375.174 -564.302
328.449 -534.157
294.0 -568.824
170.186 -588.419
224.448 -540.186
299.811 -479.895
256.1 -461.808
156.621 -523.606
54.127 -528.128
46.59 -401.518
87.287 -389.46
91.808 -467.837
161.143 -467.837
161.143 -415.083
346.536 -424.127
351.058 -283.951
435.465 -273.4
423.407 -247.777
355.58 -240.24
242.535 -146.79
198.824 -27.716
140.041 3.937
165.664 -116.644
58.648 -107.601
91.808 -172.413
177.722 -240.24
170.186 -363.836
-0.135 -366.851
2.88 -562.795
-69.469 -568.824
-134.282 -473.866
-99.614 -204.066
-61.933 -315.604
111.403 -308.067
100.852 -240.24
-19.729 -250.791
-77.005 211.939
-10.686 211.939
-12.193 -57.861
111.403 -57.861
78.243 58.198
90.301 219.476
244.042 261.679
345.029 257.157
334.478 202.896
305.84 248.114
221.433 202.896
200.331 62.72
259.115 -50.325
326.942 -154.326
345.029 -95.543
278.709 17.502
278.709 201.389
366.131 126.025
405.32 208.925
450.538 101.909
497.263 172.751
534.944 129.04
561.0 102.0
632.917 107.938
657.033 40.111
589.206 20.517
519.872 92.865
363.116 15.995
439.987 -89.514
500.277 -71.426
590.713 -35.252
666.077 -136.239
620.859 -217.631
537.959 -165.0
435.465 -185.979
518.365 -377.401
575.641 -383.43
556.046 -237.226
676.627 -314.096
676.627 -400.01
628.395 -357.807
613.322 -443.721
196.99 -197.04
