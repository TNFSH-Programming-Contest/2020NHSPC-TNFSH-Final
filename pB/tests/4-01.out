1137492.914
